packages/api-electron/index.js

```diff
@@ -108,6 +107,46 @@ const ready = (cb) => {
   app.on('ready', cb);
 };
 
+ipc.on('ssf-get-window-id', (e) => {
+  e.returnValue = BrowserWindow.fromWebContents(e.sender).id;
+});
+
+ipc.on('ssf-close-window', (e, id) => {
+  const win = BrowserWindow.fromId(id);
+  // Don't need to remove the window from the windows array, the onclose event does that for us
+  if (win) {
+    win.close();
+  }
+});
+
+ipc.on('ssf-show-window', (e, id) => {
+  const win = BrowserWindow.fromId(id);
+  if (win) {
+    win.show();
+  }
+});
+
+ipc.on('ssf-hide-window', (e, id) => {
+  const win = BrowserWindow.fromId(id);
+  if (win) {
+    win.hide();
+  }
+});
+
+ipc.on('ssf-focus-window', (e, id) => {
+  const win = BrowserWindow.fromId(id);
+  if (win) {
+    win.focus();
+  }
+});
+
+ipc.on('ssf-blur-window', (e, id) => {
+  const win = BrowserWindow.fromId(id);
```

Comment:
I have changed it to something like this:
```
ipc.on('ssf-blur-window', (e, id) => {
  getWindowFromId(id, (win) => win.blur());
});

const getWindowFromId = (id, cb) => {
  const win = BrowserWindow.fromId(id);
  if (win) {
    cb(win);
  }
};
```
Do you think this is better, or did you have something else in mind?